File: app/src/main/java/com/example/githubproject/data/repository/MainRepository.java
package com.example.githubproject.data.repository;

import com.example.githubproject.data.api.ApiHelper;

import javax.inject.Inject;

public class MainRepository {

    private ApiHelper apiHelper;

    @Inject
    public MainRepository(ApiHelper apiHelper){
        this.apiHelper = apiHelper;
    }

    public io.reactivex.Observable<retrofit2.Response<com.example.githubproject.data.model.CarsModel>> getUsers(int page){
        return apiHelper.getUsers(page);
    }

}

File: app/src/main/java/com/example/githubproject/data/repository/MainRepository.kt
package com.example.githubproject.data.repository

import com.example.githubproject.data.api.ApiHelper
import javax.inject.Inject

class MainRepository @Inject constructor(private val apiHelper: ApiHelper){
    suspend fun getUsers(_key: String, _q:String, _days:Int, _aqi:String, _alerts: String) =
        apiHelper.getUsers(_key, _q, _days,_aqi, _alerts)

    suspend fun getRegion(_key: String, _q:String) = apiHelper.getRegion(_key, _q)
}